docs/source/data/kedro_dvc_versioning.md

```diff
@@ -0,0 +1,196 @@
+# Data and pipeline versioning with Kedro and DVC
+
+This document explains how to use [DVC](https://dvc.org/), a command line tool and VS Code Extension to help you develop reproducible machine learning projects, to version datasets and pipelines in your Kedro project.
+
+## Versioning data with .dvc files
+
+### Initialising the repository
+
+For this example, we will be using a Kedro `spaceflights-pandas` starter project, which includes preconfigured datasets and pipelines.
```

Comment:
I know it's a bit annoying but I think we should tell the user here what command to run to initialise the starter, or at least refer them to the appropriate docs 😇

Reply:
For full context, this is what I did:

```
uvx kedro new --starter=spaceflights-pandas --name=test-spaceflights-dvc
```